grpc/ra-wrappers.go

```diff
@@ -122,6 +122,31 @@ func (rac RegistrationAuthorityClientWrapper) UpdateAuthorization(ctx context.Co
 	return PBToAuthz(response)
 }
 
+func (rac RegistrationAuthorityClientWrapper) PerformValidation(
```

Comment:
Generally when adding new RPCs we've tried to move away from doing conversions in the wrappers and have as much of that logic as possible happen at the call site.

Reply:
@rolandshoemaker I'm not sure I understand what conversion is happening here that you want to see moved. The `PerformValidation` wrapper calls `PerformValidation` on the inner RAC. Can you expand on what you mean?

Reply:
Chatted out of band. Will address the wrapper change to use the proto type down to the RA :+1:

Reply:
Implemented in ee82565